src/App.vue

```diff
@@ -417,7 +417,7 @@ export default {
       this.json = JSON.stringify({
         id: '1',
         to: '104222@telegram.gw.msging.net',
-        type: 'application/vnd.lime.chatstate+json',
+        type: 'application/vnd.lime.chatstater+json',
```

Comment:
Is this the unsupported type??

Reply:
Yes, this is just an example of a type that is unknown for the blipcards component